spec/dummy/config/initializers/secret_token.rb

```diff
@@ -1,7 +0,0 @@
-# Be sure to restart your server when you modify this file.
-
-# Your secret key for verifying the integrity of signed cookies.
-# If you change this key, all old signed cookies will become invalid!
-# Make sure the secret is at least 30 characters and all random,
-# no regular words or you'll be exposed to dictionary attacks.
-Dummy::Application.config.secret_token = 'f04f565f9a4db0f0af88f4bd8b79952d139b02434b889a7c7bb3fe83405e9032409bd16bca7b0d7d68a8b119b6ddfd31b17d19155cd699a27e19a48bd05eb200'
```

Comment:
Similar to the discussion in the `metasploit-concerns`, I'd recommend to replace this with an `ENV` read-out, and update the documentation to use a `.env` file with a generated value.

Reply:
Cross-referencing response: https://github.com/rapid7/metasploit-credential/pull/181#discussion_r1656814535